docs/src/main/sphinx/admin/properties-optimizer.md

```diff
@@ -54,6 +54,8 @@ This strategy will perform poorly if the number of distinct grouping keys is sma
 or for mix of distinct and non-distinct aggregations.
 `PRE_AGGREGATE` Computes distinct aggregations using a combination of aggregation
 and pre-aggregation steps.
+`SPLIT_TO_SUBQUERIES` Splits the aggregation input to independent sub-queries,
+where each subquery computes single distinct aggregation thus improving parallelism
```

Comment:
Please make changes to display the strategies as a list. Right now they are being rendered as a single block of text.